ide-support/revsaveasiosstandalone.livecodescript

```diff
@@ -998,6 +995,12 @@ private command revCopyMobileFiles pFiles, pBaseFolder, pAppBundle, pAppTarget,
          next repeat
       end if
       
+      -- If the file is an entitlement then handle appropriately
+      if tSource ends with ".xcent" then
+         put url ("binfile:" & tSource) after rCustomEntitlements
```

Comment:
Aren't entitlement files text files? Are you sure it makes sense to be loading them as binary files? If so, do you think it might be a good idea to add a comment to the source code explaining it?